lib/unleash/metrics_reporter.rb

```diff
@@ -14,7 +14,7 @@ def initialize
     end
 
     def generate_report
-      metrics = Unleash&.engine&.get_metrics()
+      metrics = Unleash.engine&.get_metrics
```

Comment:
Just confirming: Is this really the same thing in Ruby?

Reply:
Yeah, it's weird I know but it is the same thing